core/dvector.h

```diff
@@ -466,6 +467,16 @@ class PoolVector {
 		return OK;
 	}
 
+	String join(String delimiter) {
```

Comment:
(But given that other functions around here are done the same way, maybe you can disregard me)